gno.land/pkg/gnoclient/mock_test.go

```diff
@@ -30,11 +30,11 @@ func (m *mockSigner) Sign(cfg SignCfg) (*std.Tx, error) {
 	return nil, nil
 }
 
-func (m *mockSigner) Info() keys.Info {
+func (m *mockSigner) Info() (keys.Info, error) {
 	if m.info != nil {
 		return m.info()
 	}
-	return nil
+	return nil, nil
```

Comment:
It does not seem correct.

```suggestion
	return nil, errors.New("...")
```

Nope?

Reply:
Here is a suggestion change from @zivkovicmilos. When committing it, I also thought that we could ignore the returned error since this is just a mock for testing the client. @zivkovicmilos right ?